### PR TITLE
[master] Enable Radeon and Nouveau kernel modules in installer image

### PR DESCRIPTION
**Pull request by Tmcg2**
In order for AMD and Nvidia cards to display during the installer we need
to have the right kernel modules loaded. Turn on the approparite defconfig
options.

Install the approrpiate libdrm and linux firmware packages for radeon and
nvidia cards as well

OXT-1600

Signed-off-by: Tyler McGavran <mcgavrant@ainfosec.com>

### CODE REVIEW

**Comment by jean-edouard** [comment]
Built here: http://openxt-builder.ainfosec.com:8010/builders/openxt64/builds/6573

File: recipes-kernel/linux/4.19/defconfigs/openxt-installer/defconfig [hunk 1]
```diff
@@ -2415,7 +2415,7 @@ CONFIG_DRM_I2C_CH7006=y
 CONFIG_DRM_I2C_SIL164=y
 # CONFIG_DRM_I2C_NXP_TDA998X is not set
 # CONFIG_DRM_I2C_NXP_TDA9950 is not set
-# CONFIG_DRM_RADEON is not set
+CONFIG_DRM_RADEON=m
```
[the remark this answers is not in the document]
Comment (jean-edouard) [reply recipes-kernel/linux/4.19/defconfigs/openxt-installer/defconfig]: @tklengyel explained it to me but I forgot...
Btw, this PR adds 14MB to the compressed installer image.

[the remark this answers is not in the document]
Comment (tklengyel) [reply recipes-kernel/linux/4.19/defconfigs/openxt-installer/defconfig]: Building every kernel module into the kernel plus the initrd was an early idea to support UEFI and implement the SRTM chain that we was never actually put into practice.

[the remark this answers is not in the document]
Comment (jean-edouard) [reply recipes-kernel/linux/4.19/defconfigs/openxt-installer/defconfig]: Thank you @tklengyel! I guess it's fine to bring back some modules then.

**Comment by jean-edouard** [comment]
Merging soon.

**Comment by tijko** [comment]
Tested [Build-6573](http://openxt-builder.ainfosec.com:8010/builders/openxt64/builds/6573) on a **HP EliteDesk 800 GZ TWR**

With this hardware under UEFI I was able to verify,

  1. With the boot device set to onboard graphics device, install succeeds and boots
  2. With the boot device set to PCI, the installer warning appears and install succeeds but the boot device has to be switched back to onboard or startup fails.

**Pull request merged**